include/k4arecord/playback.hpp

```diff
@@ -14,11 +14,218 @@
 #include <chrono>
 #include <string>
 #include <vector>
+#include <memory>
```

Comment:
nit: sort includes alphabetically for easier merge (might not matter on this product but is probably good practice)